protocol/qrcode/connect_to_lao.json

```
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/jsonRPC.json",
+    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/connect_to_lao.json",
```

Ok, I now see what this is.

It should be the path to the file which is actaully

```suggestion
    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/qrcode/connect_to_lao.json",
```